### ISSUE TITLE
Create YEAT user manual

### ISSUE THREAD

**Issue by danejo3** [bioforensics/yeat#6]
YEAT needs a user manual. 

While YEAT is pretty simple at the moment, starting the documentation process will be helpful in the long run.

We can use `sphinx` documentation to help assist in the creation. After that, we'll need to update the `README` and have it point to YEAT's user manual.

**Comment by standage** [comment]
I agree regarding the importance of documentation. Given the early stage of this implementation, the manual will be pretty short. I'm as much of a sphinx fanboi as anyone, but I think at the moment we'd be better off using Markdown files viewed in Github for a user manual. We can revisit this as the workflow grows in scope and complexity.
